pkg/login/social/social.go

```diff
@@ -223,3 +238,35 @@ var GetOAuthProviders = func(cfg *setting.Cfg) map[string]bool {
 
 	return result
 }
+
+func readConfig(configFile string) (*setting.OAuthGroupMappingsConfig, error) {
+	if configFile == "" {
+		return nil, nil
+	}
```

Comment:
As commented earlier, suggest to move the logic if group mappings file specified before calling `readConfig`

Reply:
This change is made